contrib/completion/bash/docker

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
-# shellcheck disable=SC2016,SC2119,SC2155
+# shellcheck disable=SC2016,SC2119,SC2155,SC2206,SC2207
 #
 # Shellcheck ignore list:
 #  - SC2016: Expressions don't expand in single quotes, use double quotes for that.
 #  - SC2119: Use foo "$@" if function's $1 should mean script's $1.
 #  - SC2155: Declare and assign separately to avoid masking return values.
+#  - SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
+#  - SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```

Comment:
Disabled these for this file to ignore errors, such as;

```
In contrib/completion/bash/docker line 314:
	COMPREPLY=( $(compgen -W "$(__docker_volumes "$@")" -- "$current") )
                    ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).

In contrib/completion/bash/docker line 347:
		plugins=(${plugins[@]/$del/})
                         ^-- SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
```

Reply:
I verified: Yes, these warnings really don't make sense here.